### PR TITLE
[ETFE-3744] added NiTaxWarehouse MovementScenario

### PR DESCRIPTION
**Pull request by james-work-account**

### CODE REVIEW

File: app/models/sections/info/movementScenario/MovementScenario.scala
Comment (MJCallahanPage) [inline app/models/sections/info/movementScenario/MovementScenario.scala:1]: There is a reverse mapping for this, from `Destination Type` to `Movement Scenario` in `emcs-tfe-frontend`. That probably needs updating to cater for this? (I've also noticed, that logic in `emcs-tfe-frontend` is still missing Duty Paid logic too)

This logic also exists in Change of Destination so a PR is needed for that too.

**Review by MJCallahanPage — APPROVED** [review]
LGTM 🟢  - but PR is needed for:
- emcs-tfe-frontend
- emcs-tfe-change-destination-frontend

**Review by MJCallahanPage — APPROVED** [review]
LGTM 🟢  - note, I've not run the JTs or PTs. So as long as they pass then I think this all looks good.

File: app/models/sections/info/movementScenario/MovementScenario.scala
Comment (james-work-account) [reply app/models/sections/info/movementScenario/MovementScenario.scala:1]: I've raised a ticket to deal with CoD/emcs-tfe-frontend when a design has been finalised for those. I've also raised a separate ticket to address Duty Paid not being in emcs-tfe-frontend.

**Pull request merged**